src/popup.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="popup.css">
+  </head>
+
+  <body>
+    <div id="dgtw-status"></div>
+    <div id="show-changes" class="hide">
+        <input type="checkbox" id="show-changes-checkbox">
+        <label for="show-changes-checkbox">Show changes</label>
+    </div>
+    <div id="show-highlights" class="hide">
+        <input type="checkbox" id="show-highlights-checkbox">
+        <label for="show-highlights-checkbox">Show highlights</label>
+    </div>
+    <div>
+        <button id="dgtw-restore" class="hide">
+            Restore original content
+        </button>
```

Comment:
I'm playing with the popup on [this news story](https://www.chicagotribune.com/entertainment/ct-lilly-wachowski-glaad-media-awards-20160405-story.html) from the manual tests, and I'm finding the interaction between the button and the popup confusing. Why? Because after you restore the original content, the checkbox has no effect.

It seems like when you restore the original page content, the popup content needs to change too. It should probably say that the original content has been restored. You can't show/hide changes/highlights, so that option should be removed. You can't restore the original content again, so that option should be removed too. The only possible option to provide is to refresh the page so the content script runs again. Does this make sense?